lib/context.js

```diff
@@ -12,28 +12,20 @@ class RequestContext {
     this._next = next;
   }
 
-  get tokens() {
-    if (!this._req.session || !this._req.session.openidTokens) {
-      return undefined;
-    }
-    return new TokenSet(this._req.session.openidTokens);
-  }
-
-  set tokens(value) {
-    this._req.session.openidTokens = value;
-  }
-
   get isAuthenticated() {
     return !!this.user;
   }
 
+  makeTokenSet(tokenSet) {
```

Comment:
This lets developers create a `TokenSet` without importing the library directly.